committee-code-of-conduct/election.md

```diff
@@ -25,18 +25,28 @@ This document outlines the process, for Code of Conduct Committee (CoCC) electio
 
 ### Election process
 
-The steering committee will announce a request for nominations some time in July from any member of the following:
-* steering committee
-* members of the code of conduct committee
-* sig chair
-* sig tech lead
+The steering committee will announce a request for nominations some time in July from anyone meeting the [eligibility for candidacy](#eligibility-for-candidacy) guidelines.
 
 The steering committee will accept nominations for at least 1 week. Though the steering
 committee may choose to accept nominations for longer.  After the announced nomination
 period the steering committee will hold a private ballot election using time-limited
 [Condorcet] ranking use the [Schulze method] to break ties. The steering committee
 will publicly announce the results of the election.
 
+#### Nomination process
+
+Nominations can be collected via any means. In the past a Google form has been used to collect nominations. The form had the following questions which may be used as a template for subsequent elections:
+* Email Address (required)
+* Name of Nominee (required)
+* Email address of Nominee (required)
+* Why would you like to nominate this person for the Code of Conduct Committee? (required)
+* Does the nominee have any experience serving on an Ethics Committee or Code of Conduct Committee? (required)
+* How has this nominee demonstrated the [eligibility for candidacy](#eligibility-for-candidacy) guidelines? (optional)
+
+The nomination form must be communicated via the kdev mailing list and may also be publicized via blogs/social for the purposes of community awareness.
```

Comment:
Echoing @tashimi's comment:

> Additionally, I worry about the aspect of circulating nominations via blog/social because it may open up the process to people who are actively hostile to the goals of the committee.

I agree. Although once we send it to to the k-dev mailing list, anyone can tweet with a link to to the email...

Thinking out loud -- wdyt about adding a note about not mentioning this on social media from official accounts (kubernetes, k8scontributor twitter accounts)? Would that help? :thinking:

Reply:
I don't think it would help. I think we only have one potential option - which is to ask whomever submits the nomination be a Kubernetes org member so they are explicitly bound to the current Code of Conduct. External referrals would still be welcome, but they'd need a de facto org member sponsor to be considered. Thoughts on this? @tashimi

Reply:
Yeah, I would lean towards _not_ asking people to refrain from sharing on social media, I think some may share it with entirely good intentions to signal booth in their personal networks, I just don't want for @ kubernetes to blast it to thousands of people.

Reply:
It sounds like there are two items that need resolution here:
1. Remove the verbiage around blogs/social and leave it up to community to do that. In this case I would update line 46 to be `The nomination form must be communicated via the kdev mailing list.`. WDYT?
1. In response to @jdumars comment - Should I add a statement under the `Nomination process` section requiring the nominator to be a Kubernetes org member?

Reply:
1. lgtm! 
2. I'm on the fence on this one, one of our very productive members with excellent experience was not a member of Kubernetes org when they began

Reply:
> I'm on the fence on this one, one of our very productive members with excellent experience was not a member of Kubernetes org when they began

:100:  + ensuring only kubernetes org members fill the form is hard. We might just as well open it up..

Reply:
Org membership is not a blocking concern for me. Also +1 on the mailing list change.